jujugui/static/gui/src/app/components/lightbox/_lightbox.scss

```diff
@@ -21,35 +21,117 @@
     }
   }
 
-  &__content {
+  &__wrapper {
     position: absolute;
     top: 10%;
     left: 10%;
     bottom: 15%;
     right: 10%;
     text-align: center;
+  }
 
-    &-image {
-      background: $color-light;
-      border-radius: 2px;
-      padding: 2rem;
-      width: 100%;
-      height: 100%;
+  &__content {
+    background: $color-light;
+    border-radius: 2px;
+    padding: 2rem;
+    width: 100%;
+    height: 100%;
+    position: relative;
 
-      * {
-        max-height: 100%;
-        max-width: 100%;
+    * {
+      max-height: 100%;
+      max-width: 100%;
+    }
+  }
+
+  &__caption {
+    position: absolute;
+    bottom: -2rem;
+    left: 0;
+    right: 0;
+    text-align: center;
+    color: $white;
+  }
+
+  &__slide {
+    display: none;
+
+    &.is-active {
+      display: block;
+    }
+  }
+
+  &__navigation {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 0;
+    // This is to put the content above the navigation
```

Comment:
The navigation is outside the content...I'm not sure what this is for, can you expand plz

Reply:
Will add more comment. Bigger is better.